pkg/cfg/cfgmgr.go

```diff
@@ -163,71 +165,105 @@ func (c * CfgMgr) getRandomPassword() string {
     return string(out)
 }
 
-func (c *CfgMgr) CreateDB(serviceName, userName string) error {
+func (c *CfgMgr) CreateDB(serviceName, userName string) (bool, error) {
```

Comment:
```suggestion
func (c *CfgMgr) CreateDB(serviceName, userName string) (dbPresent bool, err error) {
```
Anyone using the function will have better understanding of what the return types represent